Sources/MapboxMaps/Style/Style.swift

```diff
@@ -180,7 +180,7 @@ public class Style {
     ///
     /// - Attention: This method is only effective with sources of `GeoJSONSource`
     /// type, and cannot be used to update other source types.
-    public func updateGeoJSONSource<T: GeoJSONObject>(withId id: String, geoJSON: T) throws {
+    public func updateGeoJSONSource(withId id: String, geoJSON: GeoJSONObject) throws {
```

Comment:
Previously, GeoJSONObject could only be Feature or FeatureCollection (or a user-defined conforming type). Let's confirm that it's acceptable for a GeoJSON source's data property to be a geometry that's not wrapped in a feature.

Reply:
@galinelle confirmed that a bare geometry is OK. MGLShape was also supported by MGLShapeSource in iOS map SDK v6._x_ and the macOS map SDK, though there’s naturally less you can do to style bare geometries versus features.